src/formats.ts

```diff
@@ -62,6 +69,21 @@ export const fullFormats: DefinedFormats = {
   "json-pointer-uri-fragment": /^#(?:\/(?:[a-z0-9_\-.!$&'()*+,;:=@]|%[0-9a-f]{2}|~0|~1)*)*$/i,
   // relative JSON-pointer: http://tools.ietf.org/html/draft-luff-relative-json-pointer-00
   "relative-json-pointer": /^(?:0|[1-9][0-9]*)(?:#|(?:\/(?:[^~/]|~0|~1)*)*)$/,
+  // the following formats are used by the openapi specification: https://spec.openapis.org/oas/v3.0.0#data-types
+  // byte: https://github.com/miguelmota/is-base64
+  byte: /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/gm,
+  // signed 32 bit integer
+  int32: validateInteger(32),
```

Comment:
I got it to work using:
```javascript
{
  type: "number",
  validate: validateInteger(32)
}
```

The problem that I have now is that the test on a non-numerical value (`"x"`) now returns true instead of false ?!?
Can I just safely remove that test ?

Reply:
to fail on non-numeric value the schema should have `type: "number"`, format on its own should not fail non-numbers.

Reply:
this is the general approach of JSON Schema that type should be explicit in the schema, it is never implied by anything else. Even your old definition would pass `{}` for example on that test, unless there is type in the schema.